pr2eus/speak.l

```diff
@@ -24,6 +25,9 @@
       (t
        (send msg :sound sound_play::SoundRequest::*say*)
        (send msg :arg (string sound))))
+    (when (string> (unix::getenv "ROS_DISTRO") "indigo")
```

Comment:
This comparison is not straightforward and leads malfunction since we on real robots use the fixed version of the `sound_play` package.

Reply:
How about chekcing if the message has `:volume` accessor  or other methods (e.g. checking md5)?
(Sorry, I cannot check they work)

Reply:
I see. Using find-method is suitable in this case?
https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/369/commits/daa1cb583b8ad031f720121d57ab1534c1ad5293#diff-26b37c5ff1eb538e9769876c2116fcc8R28

Reply:
@furushchev 
Modified. Please re-review this PR .
https://github.com/jsk-ros-pkg/jsk_pr2eus/commit/daa1cb583b8ad031f720121d57ab1534c1ad5293#diff-26b37c5ff1eb538e9769876c2116fcc8R28